app/uk/gov/hmrc/plasticpackagingtax/registration/controllers/actions/FormAction.scala

```diff
@@ -38,3 +40,4 @@ object FormAction {
 case object Unknown              extends FormAction
 case object SaveAndContinue      extends FormAction
 case object SaveAndComeBackLater extends FormAction
+case object Continue             extends FormAction
```

Comment:
Is this action/intent different from `Save and continue`? I'm not quite sure why it's needed?

Reply:
The passcode should not be saved in the registration details.. so it is just continue. As per the screenshots in the story it is just continue and there is no saveAndComeBackLater button as well